src/server/server-quiz/clients/api-constants.js

```diff
@@ -0,0 +1,58 @@
+const URL_EXAMPLE =
+  "https://opentdb.com/api.php?amount=10&category=26&difficulty=easy";
+const baseUrl = "https://opentdb.com";
+const API_QUERY = "/api.php?";
+const AMOUNT_OF_QUESTIONS = "amount=10";
+const DIFFICULTY = "&difficulty=easy";
+const FILM_CATEGORY_NUMBER = "&category=11";
+const SPORTS_CATEGORY_NUMBER = "&category=21";
+const COMPUTERS_CATEGORY_NUMBER = "&category=18";
+const CELEBRITIES_CATEGORY_NUMBER = "&category=26";
+const HISTORY_CATEGORY_NUMBER = "&category=23";
+const MUSIC_CATEGORY_NUMBER = "&category=12";
+
+const FILM_QUESTIONS_URL =
+  baseUrl + API_QUERY + AMOUNT_OF_QUESTIONS + FILM_CATEGORY_NUMBER + DIFFICULTY;
```

Comment:
FILM_QUESTIONS_URL is defined in one line, and another URL are defined in several lines, let's unify it? I prefer several lines

Reply:
I agree,
For some reason the autoformat gives me only the first one in one line

Reply:
Done , found the prettier config and works :)